core/templates/dev/head/services/GenerateContentIdService.ts

```diff
@@ -17,40 +17,48 @@
  * SubtitledHtml domain objects.
  */
 
-angular.module('oppia').factory('GenerateContentIdService', [
-  'COMPONENT_NAME_FEEDBACK', 'COMPONENT_NAME_HINT',
-  'COMPONENT_NAME_WORKED_EXAMPLE', function(
-      COMPONENT_NAME_FEEDBACK, COMPONENT_NAME_HINT,
-      COMPONENT_NAME_WORKED_EXAMPLE) {
-    var generateIdForComponent = function(existingComponentIds, componentName) {
-      var contentIdList = angular.copy(existingComponentIds);
-      var searchKey = componentName + '_';
-      var count = 0;
-      for (var contentId in contentIdList) {
-        if (contentIdList[contentId].indexOf(searchKey) === 0) {
-          var splitContentId = contentIdList[contentId].split('_');
-          var tempCount =
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
+import { AppConstants } from '../app.constants';
```

Comment:
Change this to:
`import { AppConstants } from 'app.constants';
`
Also add a newline above.

Reply:
Done